src/java/picard/reference/NormalizeFasta.java

```diff
@@ -24,7 +24,17 @@
 @CommandLineProgramProperties(
         usage = "Takes any file that conforms to the fasta format and " +
                 "normalizes it so that all lines of sequence except the last line per named sequence " +
-                "are of the same length.",
+                "are of the same length.  First convert bam file to FASTQ." +
```

Comment:
> First convert bam file to FASTQ

I don't understand what this means. Does the input need to be fastq, or does the program do this internally?